docs/source/examples/cdr_qrack.md

```diff
@@ -0,0 +1,178 @@
+
+```{code-cell}
+# Use Qiskit's Fave5QV1 as a noisy simulator
+def qiskit_noisy(circuit: cirq.Circuit, shots=1000):
+    """Execute the input circuit and return the expectation value of |00..0><00..0|"""
+
+    # Cirq -> Qiskit circuit
+    qiskit_circ = mitiq.interface.mitiq_qiskit.to_qiskit(circuit)
+
+    # Add measurement gates to the circuit
+    qiskit_circ.measure_all()
+
+    # Setup the fake backend and run the circuit
+    noisy_backend = Fake5QV1()
+    job = noisy_backend.run(qiskit_circ, shots=shots)
+
+    # Use the resulting counts to return the expectation value of 00
+    counts = job.result().get_counts()
+    ret_val =  counts[qiskit_circ.num_qubits * "0"] / shots
+    return ret_val
+```
+
+## Cirq Simulator for exact result
+
+The `compute_density_matrix` is the Cirq density matrix simulator with a Mitiq wrapper. It is used to obtain the exact `00` expectation value. This is then used to determine the accuracy of the mitigated and unmitigated reuslts.
+
+```{code-cell}
+def cirq_simulate(circuit: cirq.Circuit) -> np.ndarray:
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit 
+    executed without depolarizing noise.
+    """
+    res = compute_density_matrix(circuit, noise_level=(0.0,))
+    return res[0, 0].real
+```
+
+## Executing CDR
+
+With the different executor functions defined for running the Qrack, Qiskit, and Cirq simulators, the `mitiq.cdr.execute_with_cdr` function can now be called.
+
+```{code-cell}
+ideal_expval = cirq_simulate(circuit).round(5)
+print(f"Ideal expectation value from Cirq Simulator: {ideal_expval:.3f}")
+
+unmitigated_expval = qiskit_noisy(circuit)
+print(f"Unmitigated expectation value from Qiskit Fake backend: {unmitigated_expval:.3f}")
+
+mitigated_expval = cdr.execute_with_cdr(
+    circuit,
+    qiskit_noisy,
+    simulator=qrack_simulate,
+    random_state=0,
+)
+print(f"Mitigated expectation value with Mitiq CDR: {mitigated_expval:.3f}\n")
+
+```
+
+## Conclusion
+By learning from the near-Clifford circuits that resemble the original circuit, CDR is able to apply the noise information learned to the original circuit. From the example, you can see that the mitigated expectation value is closer to the ideal value.
+
+The improvement factor with CDR can be seen below to highlight the mitigated results that can be provided with this technique.
+
+```{code-cell}
+unmitigated_error = abs(unmitigated_expval - ideal_expval)
+mitigated_error = abs(mitigated_expval - ideal_expval)
+
+print(f"Unmitigated Error:           {unmitigated_error:.3f}")
+print(f"Mitigated Error:             {mitigated_error:.3f}")
+
+improvement_factor = unmitigated_error / mitigated_error
+print(f"Improvement factor with CDR: {improvement_factor:.2f}")
+```
```

Comment:
Let's just add one more thing to the end here. Something like "If you want to learn more about CDR check out the original reference and our user guide."

Reply:
I added a section at the very end including links to the other CDR info and also how to contact us with questions.